### PR TITLE
[ET-1980] - Fix My Tickets Update Button When Only Opt-Out Form Shows

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1980] 

### 🗒️ Description

We found that when only Tickets appear on the My Tickets page, the update button only shows when IAC or ARF are in place. The update button does NOT show in the case where the user can opt attendees out of the public attendees list.

This PR adds that logic in the correct place.

### 🎥 Artifacts 

image

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1980]: https://stellarwp.atlassian.net/browse/ET-1980?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -204,6 +204,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 * Fix - Resolved an issue where Order Status was not populated when exporting the Attendee List using Tickets Commerce. [ET-1883]
 * Tweak - Update default footer text of Tickets Emails to include link to website. [ET-1971]
 * Fix - Update usage of `method_exists()` to comply with PHP 8.1 standards. [ET-1759]
+* Fix - Update button will now show when the opt-out checkbox shows ont heMy Tickets page. [ET-1980]
```
Comment (redscar) [inline readme.txt:207]:
```suggestion
* Fix - Update button will now show when the opt-out checkbox shows on the My Tickets page. [ET-1980]
```

File: src/views/tickets/orders.php [hunk 2]
```diff
@@ -50,6 +50,9 @@
 $user_has_rsvp              = $rsvp->get_attendees_count_going_for_user( $event_id, $user_id );
 $tribe_my_tickets_have_meta = false;
 
+// Check to see if we are hiding the opt-out form.
+$hide_attendee_list_optout = apply_filters( 'tribe_tickets_plus_hide_attendees_list_optout', false, $event_id );
```
Comment (redscar) [inline src/views/tickets/orders.php:54]: Do we need to add a docblock?

**Review by redscar — APPROVED** [review]

File: readme.txt [hunk 3]
```diff
@@ -205,6 +205,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 * Fix - Resolved an issue where Order Status was not populated when exporting the Attendee List using Tickets Commerce. [ET-1883]
 * Tweak - Update default footer text of Tickets Emails to include link to website. [ET-1971]
 * Fix - Update usage of `method_exists()` to comply with PHP 8.1 standards. [ET-1759]
+* Fix - Update button will now show when the opt-out checkbox shows on the My Tickets page. [ET-1980]
```
Comment (estebanhdzc) [inline readme.txt:208]: does the new filter `tribe_tickets_plus_hide_attendees_list_optout` need to appear also in the readme?

Comment (estebanhdzc) [reply readme.txt:208]: Great, thanks for clarification @codingmusician !

**Review by estebanhdzc — APPROVED** [review]

**Pull request merged**